cg/cli/archive.py

```diff
@@ -82,3 +82,48 @@ def delete_file(context: CGConfig, dry_run: bool, file_path: str):
         data_flow_config=context.data_flow,
     )
     spring_archive_api.delete_file(file_path=file_path, dry_run=dry_run)
+
+
+@archive.group("retrieve-spring")
+def retrieve_spring():
+    """Retrieve spring files."""
+    pass
+
+
+@retrieve_spring.command("sample")
+@click.pass_obj
+@click.argument("sample_id", required=True)
+def retrieve_spring_sample(context: CGConfig, sample_id: str):
```

Comment:
Maybe all of these should have `for` in them. What do you think?
```suggestion
def retrieve_spring_for_sample(context: CGConfig, sample_id: str):
```

Reply:
Makes sense given the names in the SpringArchiveAPI as well 💪